website_sale_product_subscribe_notify/models/product_template.py

```diff
@@ -24,14 +24,16 @@ def write(self, vals):
                     partner_ids = rec.get_product_category_followers_ids()
                     partners = rec.env['res.partner'].browse(partner_ids)
                     list_price = "%d" % int(rec.list_price)
-                    for partner in partners:
+                    limit_recipient = rec.env['ir.default'].get('res.config.settings', 'email_recipient_limit')
+                    number_of_loop = len(partners)/limit_recipient
+                    if number_of_loop%1 != 0 : number_of_loop += 1 
```

Comment:
Maybe we can use the `math` lib? 
number_of_loop = math.ceil(len(partners)/float(limit_recipient))

Reply:
so by using math.ceil(), I will remove the if condition then.